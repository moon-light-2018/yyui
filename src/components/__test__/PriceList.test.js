import React from 'react';
import { shallow } from 'enzyme'
import PriceList from '../PriceList'
import { items, categoryData } from '../../testData'
const itemsWithCategory = items.map(item => {
    item.category = categoryData[item.cid]
    return item
})
const props = {
    items: itemsWithCategory,
    onModifyItem: () => { },
    onDeleteItem: () => { }
}
let wrapper
describe('test priceList', () => {
    beforeEach(() => {
        wrapper = shallow(<PriceList {...props} />)
    })
    it('should render the component snapshot', () => {
        expect(wrapper).toMatchSnapshot()
    })
    it('should render correct', () => {
        expect(wrapper.find('.list-group-item').length).toEqual(itemsWithCategory.length);
    });
})

