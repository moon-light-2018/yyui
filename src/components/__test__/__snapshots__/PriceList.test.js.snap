// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test priceList should render the component snapshot 1`] = `ShallowWrapper {}`;
